src/Bravo3/Orm/Config/Configuration.php

```diff
@@ -8,15 +8,9 @@ class Configuration
      */
     protected $cache_dir;
 
-    /**
-     * @var bool
-     */
-    protected $hydration_exceptions_as_events;
-
-    public function __construct($cache_dir = null, $hydration_exceptions_as_events = false)
+    public function __construct($cache_dir = null)
     {
-        $this->cache_dir                      = $cache_dir;
-        $this->hydration_exceptions_as_events = $hydration_exceptions_as_events;
```

Comment:
Ahh, forgot about that one. And yes, but I'm pretty sure all it needs to do is drop off the second parameter. I don't believe Phoenix relies on that functionality any more.